cmd/helm/init_test.go

```diff
@@ -202,6 +203,35 @@ func TestEnsureHome(t *testing.T) {
 		t.Error(err)
 	}
 
+	rr, err := repo.LoadRepositoriesFile(hh.RepositoryFile())
+	if err != nil {
+		t.Error(err)
+	}
+
+	foundStable := false
+	for _, rr := range rr.Repositories {
```

Comment:
Nit: `rr` is shadowing the `rr` from above, I recommend changing the variable name here

Reply:
changed the other one; there are a non-zero number of occurrences of `rf, err := repo.LoadRepositoriesFile` elsewhere.